doc/manual/spec

```diff
@@ -214,24 +214,67 @@ can express this as
 	BuildConflicts: gcc <= 2.7.2.1
 \endverbatim
 
+\section comments Comments
+
+Comments in spec file have # at the start of the line.
+\verbatim
+	# this is a comment
+\endverbatim
+Macros are expanded in comments, thus if a macro is in comment use two percent
+characters before it:
```

Comment:
I'd make that more like:
```
Macros are expanded even in comment lines. If this is undesireable, escape the macro with an extra percent sign (%):
<...example..>
```

Now that we have such a thing, it'd be good to mention %dnl as a less quirky way of adding comments to specs.